test/sourcemap.coffee

```diff
@@ -63,3 +64,55 @@ test "#3075: v3 source map fields", ->
   v3SourceMap = JSON.parse v3SourceMap
   arrayEq v3SourceMap.sources, ['tempus_fugit.coffee']
   eq v3SourceMap.sourceRoot, './www_root/coffee/'
+
+# Source maps aren't accurate on Node v12 ??
+if process.version.split(".")[0] != "v12"
+  test "native source maps", ->
+    new Promise (resolve, reject) ->
+      proc = spawn "node", [
+        "--enable-source-maps"
+        "-r", "./register.js"
+        "-r", "./test/integration/error.coffee"
```

Comment:
All of our test “fixture” files are in `test/importing`. Do you mind moving `error.coffee` there?

```suggestion
        '--require', './register.js'
        '--require', './test/importing/error.coffee'
```

And some style notes: I prefer to spell out command-line flags so that people know what I’m using without looking it up. And I haven’t been going back and fixing this for old code, but for new code we’ve been trying to follow the rule of single-quote strings unless there’s interpolation, or when following an established pattern (like the argument to `test`); and no single-character variable names.